### DISCUSSION TITLE
iOS beta full

### DISCUSSION THREAD

**Discussion by Desy10140** [advplyr/audiobookshelf-app#873] [General]
Is it possible to make more room for new iOS testers?

**Comment by DieselTech** [comment]
No. That's controlled by apple and they don't allow exceptions.

**Reply by hobesman** [reply]
What would be the implications of taking it out of beta? I realize it might lead to some negative reviews from people expecting a more polished product. And maybe that's reason enough. But what other effects does it have? I'm not on iOS so I'm not pushing either way. Just curious.

**Reply by nichwall** [reply]
It's mostly that it isn't as polished as it could be, as well as iOS having more bugs since there are less contributors who knows how to work with the iOS version. There's a GH issue somewhere keeping track of things to fix first.

Testflight also allows crash reports to be automatically submitted so it's a lot easier to get feedback.

**Reply by DieselTech** [reply]
> What would be the implications of taking it out of beta?

Held to higher standards and having to pay the apple dev fee. The higher standards mean that if the app has too many bugs that surface while the apple devs are testing it, they will just deny the app from being listed on the app store. Trying to get an app that failed verification to go back and get passed is annoying and time consuming. So it's better to make sure things are as polished as possible before submitting.

**Reply by TrekScout** [reply]
Is there a ballpark ETA for the iOS release?

**Reply by nichwall** [reply]
0.9.66 should already be available on testflight

**Reply by benparsell** [reply]
How do new users go about gaining access to this? What in the world is the point of have a publically available platform that is inaccessible through one of the primary interfaces (iOS)?

**Reply by DieselTech** [reply]
> How do new users go about gaining access to this? What in the world is the point of have a publically available platform that is inaccessible through one of the primary interfaces (iOS)?

Take your grievances up with apple. These are the limitations they impose on apps. You can sideload through AltStore/SideStore if you want in that bad. 

Or get an android phone. They don't have these limits and android users can use the app free of limits from the play store.

**Reply by nichwall** [reply]
There is still not an estimate on a full iOS release (since the iOS app isn't the only part of a project which is mostly done by a single dev). "Soon", but not an actual estimate of a release date.
https://www.audiobookshelf.org/faq#app-faq
